spec/controllers/api/v1/patients_controller_spec.rb

```diff
@@ -0,0 +1,165 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::PatientsController, type: :controller do
+  describe 'POST sync' do
+    it 'creates new patients' do
+      patients = (1..10).map { build_patient }
+      post(:sync_from_user, params: { patients: patients })
+      expect(Patient.count).to eq 10
+      expect(Address.count).to eq 10
+      expect(PhoneNumber.count).to eq(patients.sum { |patient| patient['phone_numbers'].count })
+      expect(response).to have_http_status(200)
+    end
+
+    it 'creates new patients without address' do
```

Comment:
Should we test for creating patients without a phone number?

Reply:
Yeah, added in 5c5d8fc.